uefi.adoc

```diff
@@ -19,8 +19,9 @@ IMPORTANT: All content in this section is optional and recommended for BRS-B.
 | `UEFI_030` a| MUST meet the following memory map rules:
 
               * The default memory space attribute is `EFI_MEMORY_WB`.
-              * Address translation is enabled.
+              * Paged virtual-memory scheme MUST be configured with identity mapping to support memory protection.
               * Only use `EfiRuntimeServicesData` memory type for describing any SMBIOS data structures.
+2+| _Enabling paged virtual memory scheme means satp.MODE must not be set to `Bare` and proper page table for identity mapping is setup by the firmware. This is required for platform protection use cases before handing off to the OS._
```

Comment:
s/proper page table for identity mapping/proper page table with identity mapping/

I think the page table is setup by OS and this page table identity mapping is required when OS makes EFI call at boot-time or run-time.

Reply:
> s/proper page table for identity mapping/proper page table with identity mapping/
> 
Thanks!. Let me update.